index.html

```diff
@@ -13,16 +13,16 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>My-Portfolio</title>
+    <title>Roshan Awesome Portfolio</title>
     <link rel="preconnect" href="https://fonts.googleapis.com">
 <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
 <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
     <nav class="navbar">
-      <a href="#">My Logo</a>
-      <img class="menu" src="Union.png" alt="Image of vector">
+      <a href="#">Roshan Bajgain</a>
+      <img class="menu" src="Union.png" alt="Menu">
       <div class="menu-items">
         <ul class="nav1">
           <li>portfolia</li>
```

Comment:
- [ ] [OPTIONAL] For all your nav links you should add `a` tag, check the next sample:

```
<ul>
  <li><a href="#">Nav Link 1</li>
  <li><a href="#">Nav Link 2</li> 
  <li><a href="#">Nav Link 3</li>
</ul>
```